<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameHistory</class>
 <widget class="QWidget" name="gameHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newGameButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>New game</string>
   </property>
  </widget>
  <widget class="QListWidget" name="gameList">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>500</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Main menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="switchPlayerButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch player</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
